Futilities/Futilities.FileIO.Tests/FileIOChecksumTests.cs

```diff
@@ -16,11 +16,12 @@ namespace Futilities.FileIO.Tests
     [TestClass]
     public class FileIOChecksumTests : TestingBase
     {
-        private string GetProjectDirectory => Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.FullName;
+        //private string GetProjectDirectory => Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.FullName;
```

Comment:
```suggestion
```

Reply:
This is replaced completely in the next PR